infrastructure-playbooks/rolling_update.yml

```diff
@@ -91,6 +91,7 @@
         - not containerized_deployment
 
   roles:
+    - ceph-defaults
```

Comment:
You're gonna need either ``ceph-common`` or ``ceph-docker-common`` here depending on the value of ``containerized_deployment``.  Same for the rest of the plays in this playbook.